src/tests/unit/tests/DetailsView/components/issue-filing-dialog.test.tsx

```diff
@@ -40,7 +40,7 @@ describe('IssueFilingDialog', () => {
         };
         deps = {
             bugFilingServiceProvider: null,
```

Comment:
Might be better to pass something other than null here.

Reply:
The reason I passed as null is because we don't care about what the actual deps are, just that they are the same object passed into children. Hence, if deps was {} or null, I would agree that is not okay but I'm fine with this.